docs/client-sdk/javascript/tutorials/build-an-xmtp-hello-world-app.md

```diff
@@ -0,0 +1,576 @@
+   import { mainnet, polygon } from "wagmi/chains";
+
+   const chains = [ mainnet, polygon];
+   // Wagmi client
+   const { provider } = configureChains(chains, [
+     walletConnectProvider({ projectId: "<YOUR_PROJECT_ID>" }),
+   ]);
+   const wagmiClient = createClient({
+     autoConnect: true,
+     connectors: modalConnectors({ appName: "web3Modal", chains }),
+     provider,
+   });
+   // Web3Modal Ethereum Client
+   const ethereumClient = new EthereumClient(wagmiClient, chains);
+
+   function App() {
+     // highlight-next-line
+     const { isConnected } = useAccount()
+
+     return (
+       <div className="App">
+         <Web3Button/>
+         <Web3Modal
+           projectId="<YOUR_PROJECT_ID>"
+           ethereumClient={ethereumClient}
+         />
+         // highlight-next-line
+        {isConnected && <button onClick={initXmtp}>Connect to XMTP</button>}
+       </div>
+     );
+   }
+
+   export default App;
+   ```
+   So now, the **Connect to XMTP** button should only show when your wallet is connected.
+   
+3. Now, let's add an onClick handler when users click the button. This will initialize XMTP with your app.
+   ```tsx title="App.tsx" showLineNumbers
+   import React from 'react';
+   import './App.css';
+   import {
+     EthereumClient,
+     modalConnectors,
+     walletConnectProvider,
+   } from "@web3modal/ethereum";
+   import { Web3Modal, Web3Button } from "@web3modal/react";
+   import { configureChains, createClient,} from "wagmi";
+   import { mainnet, polygon } from "wagmi/chains";
+   // highlight-start
+   import { fetchSigner } from '@wagmi/core'
+   import { Client } from '@xmtp/xmtp-js'
+   // highlight-end
+   
+   const chains = [ mainnet, polygon];
+   // Wagmi client
+   const { provider } = configureChains(chains, [
+     walletConnectProvider({ projectId: "<YOUR_PROJECT_ID>" }),
+   ]);
+   const wagmiClient = createClient({
+     autoConnect: true,
+     connectors: modalConnectors({ appName: "web3Modal", chains }),
+     provider,
+   });
+   // Web3Modal Ethereum Client
+   const ethereumClient = new EthereumClient(wagmiClient, chains);
+
+   function App() {
+     const { isConnected } = useAccount()
+     // highlight-start
+     const initXmtp = async function() {
+       const signer = await fetchSigner()
+       const xmtp = await Client.create(signer, { env: "production" })
+     }
+     // highlight-end
+     return (
+       <div className="App">
+         <Web3Button/>
+         <Web3Modal
+           projectId="<YOUR_PROJECT_ID>"
+           ethereumClient={ethereumClient}
+         />
+        {isConnected && <button onClick={initXmtp}>Connect to XMTP</button>}
+       </div>
+     );
+   }
+
+   export default App;
+   ```
+   At this point, clicking the **Connect to XMTP** button will ask your wallet for a signature request. Below is an example of a MetaMask Signature request screen for signing to enable an XMTP identity.
```

Comment:
I don't see the screen below. Do you want to include the screen or just remove this sentence? =)

Reply:
Added image. Thanks for catching this!